cmd/buildkitd/util.go

```diff
@@ -27,20 +27,16 @@ func parseIdentityMapping(str string) (*idtools.IdentityMapping, error) {
 		return nil, errors.Errorf("user namespaces are only supported on linux")
 	}
 
-	idparts := strings.Split(str, ":")
+	idparts := strings.SplitN(str, ":", 3)
 	if len(idparts) > 2 {
 		return nil, errors.Errorf("invalid userns remap specification in %q", str)
 	}
 
 	username := idparts[0]
-	groupname := username
-	if len(idparts) == 2 {
-		groupname = idparts[1]
-	}
 
-	logrus.Debugf("user namespaces: ID ranges will be mapped to subuid/subgid ranges of: %s:%s", username, groupname)
+	logrus.Debugf("user namespaces: ID ranges will be mapped to subuid ranges of: %s", username)
 
-	mappings, err := idtools.NewIdentityMapping(username, groupname)
+	mappings, err := idtools.NewIdentityMapping(username)
```

Comment:
Just ignoring the `:gid`  now; I see UserNS is still for testing only; https://github.com/moby/buildkit/commit/6921dbe0f6a6e276c999000b826c316e62a4bae3

wondering if the configuration should only accept `uid` (and produce error if `:gid` is specified), or if we should continue the current approach (accept both, but ignore `gid`) ?